### PR TITLE
Add navigation

### PR DESCRIPTION
**Pull request by Gw3i**

### CODE REVIEW

**Comment by j-eick** [comment]
Good job 👍🏼  One thing... ⬇️

File: src/styles.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const GlobalStyle = createGlobalStyle`
 
 	body {
 		margin: 0;
```
Comment (j-eick) [inline src/styles.js:15]: Are you overwriting something here or could you actually delete **margin** and **padding**? (for the sake of cleaner code)

Comment (Gw3i) [reply src/styles.js:15]: Yes, thanks! There was a preset of margin, but no padding

**Review by j-eick — APPROVED** [review]
Nicely done.  👍🏼

File: src/components/Nav.jsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ export default function Nav() {
 					<Link href="/">
 						<StyledNavLink>
 							<Icon
-								size="32"
+								size="32px"
```
Comment (JessicaLoers) [inline src/components/Nav.jsx:19]: 👍

File: src/components/Nav.jsx [hunk 3]
```diff
@@ -27,7 +27,7 @@ export default function Nav() {
 					<Link href="/create">
 						<StyledNavLink>
 							<Icon
-								size="32"
+								size="32px"
```
Comment (JessicaLoers) [inline src/components/Nav.jsx:30]: 👍

**Review by JessicaLoers — APPROVED** [review]
**Hi @Gw3i**  looks good to me.

**Pull request merged**